RubeesEat/Model/DB/DbPersonRepository.cs

```diff
@@ -0,0 +1,92 @@
+using System.Data;
+
+namespace RubeesEat.Model.DB;
+
+public class DbPersonRepository(IDbConnectionFactory connectionFactory) : IPersonRepository
+{
+    public IReadOnlyList<Person> GetAll()
+    {
+        var persons = new List<Person>();
+
```

Comment:
Please declare this variable closer to the usage. Right now, the declaration looks out of place. We should place it just above the `while` loop below.